### DISCUSSION TITLE
Transform coordinate points for all entities inside DXF

### DISCUSSION THREAD

**Discussion by santiago-vargas-de-kruijf** [mozman/ezdxf#922] [General]
Hello,

Is it possible to transform the CRS of all entities inside an DXF?
Currently, we're implementing this process manually: We read each entity, extract its points/coordinates (In case they exist), transform them and finally create a new entity with the transformed coordinates. 

Is there a better way of doing this?

Thank you in advance.

**Comment by mozman** [comment]
The next version of `ezdxf` v1.1 will have a `transform` module: https://ezdxf.mozman.at/docs/transform.html

The `transform.inplace()`transforms the given entities inplace by a transformation matrix and the `transform.copies()` function copies the given entities and optionally transform these copies.

**Reply by santiago-vargas-de-kruijf** [reply]
Thank you for the fast reply. We will look into this.
